onnxruntime/core/mlas/lib/intrinsics/avx2/qladd_avx2.cpp

```diff
@@ -75,6 +75,30 @@ MlasPackS16_256<int8_t>(
     return _mm256_packs_epi16(a, b);
 }
 
+MLAS_FORCEINLINE
+static
+__m256i
+MlasLoad32Bytes(const uint8_t* buffer, int64_t N)
```

Comment:
This shouldn't need to be int64_t, but it can stay for now. MLAS only deal with size_t as that is the native sized value for a register.

Reply:
Yeah.